web-app/src/components/SettingsDialog.js

```diff
@@ -478,12 +478,44 @@ class SettingsDialog extends React.Component {
 									style={{
 										color: "rgba(40, 26, 26, 0.56)",
 										marginTop: "16px",
-										padding: "16px 0px",
+										padding: "16px 0px 0px 0px",
 									}}
 								>
-									Custom nation colours
+									Nation colours
+								</Typography>
+								<div width="100%">
+									<FormControlLabel
+										style={{
+											width: "100%",
+											maxWidth: "920px",
+											paddingLeft: "0px",
+										}}
+										classes={{
+											root: classes.formControlLabel
+										}}
+										control={
+											<Switch
+// TODO: Add switch logic here
+//												checked={{}}											
+//												onChange={}
+											/>
+										}
+										label="Colour non-SC provinces"
+									/>
+									<Typography variant="caption" style={{color: "rgba(0,0,0,0.56)"}}>
+									Colour provinces without supply centers based on the ownership of the supply centers around it.
+									</Typography>
+								</div>
+								<Typography
+									variant="subtitle2"
+									style={{
+										color: "rgba(40, 26, 26, 0.56)",
+										marginTop: "8px",
+										padding: "16px 0px 8px 0px",
+									}}
+								>
```

Comment:
Could you use the `style` object at the top of the script. Inline styling is the devil and we should move away from it.

See how it is done in other parts of the app and you should be able to work it out.

In the style object we should avoid using hard-coded values, e.g. "16px 0px 8px 0px". Material ui has a cool spacing feature which we should use instead: `padding: theme.spacing(2, 0, 1, 0);`
The spacing function works using an 8pt scale so that translates to the same value that you have.

Similarly, we shouldn't use hard-coded colors. If you want grey you should use `theme.palette.grey[500]`. Note that the 500 here could be wrong. Try experiment with other numbers in the range [50, 100, 200, 300, 400, 500, 600, 700, 800, 900] to get a grey that matches the other greys on the page.

The reason that all of this is important is because:
(1) If we decide we want to change the color or padding of every subtitle on the page we can do it in the one place
(2) If we decide to change from 8pt spacing to 10pt spacing we can do that in one place and it will take effect across the entire app
(3) We can easily (very easily!) introduce themes, e.g. light mode dark mode

Reply:
![image](https://user-images.githubusercontent.com/7293081/132376712-6a68a712-db0a-46dd-ba89-8c5fac8f325c.png)
I would love to use the style object, but could you help me explain how it works?

There are default material spacings that I try to use, however going with the flow of the page (e.g. in this case, the 'Colours' and 'Variant' being way too close together), it's sometimes needed to make adjustments. Using theme.spacing gives me a noncompile.

For this build, I think it's super good - but like the other moment; save it for after this 'major' (for me) feature release. Still good comments, I'll save them.

Reply:
Boyscouted subtitle2